tests/ui/linkage-attr/raw-dylib/elf/verbatim.rs

```diff
@@ -0,0 +1,25 @@
+//@ only-elf
```

Comment:
The `x86_64-fortanix-unknown-sgx` target doesn't support dynamic linking. Can you please add `//@ ignore-sgx` to ignore this platform?